docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala

```diff
@@ -17,15 +17,15 @@ object MyExplicitExceptionHandler {
   import StatusCodes._
   import Directives._
 
-  object MyApp extends App {
+  val myExceptionHandler = ExceptionHandler {
+    case _: ArithmeticException =>
+      extractUri { uri =>
+        println(s"Request to $uri could not be handled normally")
+        complete(HttpResponse(InternalServerError, entity = "Bad numbers, bad result!!!"))
+      }
+  }
 
-    val myExceptionHandler = ExceptionHandler {
```

Comment:
This was written like this by purpose to allow copy/pasting the docs.

Reply:
As far as i remember the problem with previous implementation was that App class is extending a DelayedInit. Because of that in the both tests handlers are not initialized. e.g MyExplicitExceptionHandler.MyApp.myExceptionHandler is null.

Reply:
@jrudolph if you want to have it copy paste-able, maybe the whole snippet can be wrapped in an `object`. As `App` extends `DelayedInit` it causes line 82 to blow with an exception as it's not initialized.

Reply:
I see. Maybe then it's ok the way it is now.